bucket/streamdeck.json

```
@@ -0,0 +1,31 @@
+{
+    "homepage": "https://www.elgato.com/en/gaming/downloads#sd-win",
+    "description": "Stream Deck lets you take control of your content and focus on what's important: your audience.",
+    "version": "4.3.3.11845",
+    "license": "Freeware",
+    "architecture": {
+        "64bit": {
+            "url": "https://edge.elgato.com/egc/windows/sd/Stream_Deck_4.3.3.11845.msi",
+           "hash": "c458695aaff4d34bbe6a56dce1caf371fc9943036dea137bf4daaeb7fd702883"
```

One last aestetic change, which is triggering me.

```suggestion
            "hash": "c458695aaff4d34bbe6a56dce1caf371fc9943036dea137bf4daaeb7fd702883"
```